bodhi/tests/server/functional/test_admin.py

```diff
@@ -0,0 +1,44 @@
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+import copy
+from webtest import TestApp
+
+from bodhi.server import main
+from bodhi.server.models import (User, Group)
```

Comment:
Actually, it turns out that you also don't need this import and it is causing one of the tests to fail:

```
/home/vagrant/bodhi/bodhi/tests/server/functional/test_admin.py:19:1: F401 'bodhi.server.models.Group' imported but unused
/home/vagrant/bodhi/bodhi/tests/server/functional/test_admin.py:19:1: F401 'bodhi.server.models.User' imported but unused
F.
======================================================================
FAIL: Enforce PEP-8 compliance on the codebase.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/vagrant/bodhi/bodhi/tests/test_style.py", line 37, in test_code_with_flake8
    self.assertEqual(subprocess.call(flake8_command), 0)
AssertionError: 1 != 0
```